python/tvm/driver/tvmc/config_options.py

```diff
@@ -21,8 +21,47 @@
 """
 import os
 import json
+
+from tvm._ffi import libinfo
 from tvm.driver.tvmc import TVMCException
 
+CONFIGS_JSON_DIR = None
+
+
+class ConfigsJsonNotFoundError(TVMCException):
+    """Raised when the JSON configs dirtree cannot be found."""
+
+
+def get_configs_json_dir() -> str:
+    """Find the 'configs' directory, containing the JSON files used to configure tvmc
+    with persistent argument settings.
+
+    Returns
+    -------
+    str :
+        The path to the 'configs' directory
+    """
+    global CONFIGS_JSON_DIR
+    if CONFIGS_JSON_DIR is None:
+        candidate_paths = []
+        candidate_paths.extend(libinfo.find_lib_path())
+        # When running from source, the configs directory will be located one directory above the
+        # native libraries, so covering that case.
+        candidate_paths.extend(
+            [os.path.abspath(os.path.join(lib_path, "..")) for lib_path in libinfo.find_lib_path()]
+        )
+        for path in candidate_paths:
+            print(f"{path}")
```

Comment:
@leandron I just spotted this rogue `print` 👻  can you remove it? I imagine it will always print this path when you run tvmc?

Reply:
🤦